sale_comment_template/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import test_sale_order_report
```

Comment:
A blank line is needed

Reply:
Done

Reply:
Thanks @SimoRubi 

@Yajo can you retest?